src/gsi/read_gfs_ozone_for_regional.f90

```diff
@@ -197,6 +210,71 @@ subroutine read_gfs_ozone_for_regional
      idate4(2)= sighead%idate(2)
      idate4(3)= sighead%idate(3)
      idate4(4)= sighead%idate(4)
+
+!  add netcdf code here
+   else if (use_gfs_ncio) then
+      atmges = open_dataset(filename)
+      ! get dimension sizes
+      ncdim = get_dim(atmges, 'grid_xt'); gfshead%lonb = ncdim%len
+      ncdim = get_dim(atmges, 'grid_yt'); gfshead%latb = ncdim%len
+      ncdim = get_dim(atmges, 'pfull'); gfshead%levs = ncdim%len
+      nsig_gfs=gfshead%levs
+      ! hard code jcap,idsl,idvc
+      gfshead%jcap = -9999
+      gfshead%idsl= 1
+      gfshead%idvc = 2
+
+      ! FV3GFS write component does not include JCAP, infer from DIMY-2
+      if (njcap<0) njcap=latb-2
```

Comment:
same question as before, what is njcap set to previously? Is this if needed?

Reply:
Removed the if.